src/commands/cleanup.ts

```diff
@@ -189,7 +189,10 @@ export default class extends Command {
                         }
                     }
                 )
-                output.push(...response.data.items)
+                // do not push empty items
+                if(Object.keys(response.data.items).length) {
```

Comment:
Shouln't this be
```suggestion
                if(Object.keys(response.data.items).length > 0) {
```
?